lambda-create.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```

Comment:
`zip -r aws_fizzbuzzx.zip .`?

Reply:
I put the comment right above the zip file call? Should i have had put it below the shebang?

Reply:
to automate the process, we need the command so the script can do it.

Reply:
though a comment with the link to the documentation is a great addition .

Reply:
OH! okay sorry misunderstanding will add ASAP